### ISSUE TITLE
MIP display, EFIS Control Panel

### ISSUE THREAD

**Issue by DeputyLOL** [Alexustas/Props-requests#2]
Thanks, sorry about not using the forum. My school internet blocks me from posting on it. :(

Do you think you could make a more realistic screen with a separate EFIS Control Panels for this monitor? And another with no buttons at all?

![image](https://cloud.githubusercontent.com/assets/12616126/10220569/8370ff40-6840-11e5-9258-98adc1c47658.png)
 Ignore the MTRS button, BARO knob, the middle right selector knob (ND range selector), right VOR switch and WXR button.
 The MINS knob would be used to choose an altitude and have a button in the middle for selecting it. 

 Secondly, LED number displays. Like the ones shown below. 
![image](https://cloud.githubusercontent.com/assets/12616126/10220638/e1db409a-6840-11e5-96db-a3c2caeec5b8.png)

 I can handle all the variables and persistent, but I cant make props...

 The last big thing for now would be animated rudder pedals.

**Comment by DeputyLOL** [comment]
Another request:
Two variants of the switch with cover. 
One with no label or light so it is much shorter.
And one with no cover, label or light.

Do you think that is possible?

**Comment by Alexustas** [comment]
Please, make separate Issue for switches

**Comment by DeputyLOL** [comment]
Done. :)

**Comment by Alexustas** [comment]
Display

![bandicam 2015-10-12 14-32-53-533](https://cloud.githubusercontent.com/assets/6713671/10424185/d6da1746-70ee-11e5-92c4-ee33eedda51a.jpg)

**Comment by DeputyLOL** [comment]
If possible, for the PFD. No buttons.

**Comment by Alexustas** [comment]
it's not a button.  it's kind of screw
![bandicam 2015-10-12 15-12-25-866](https://cloud.githubusercontent.com/assets/6713671/10424761/bb72fae0-70f3-11e5-8860-5e3e8afe3524.jpg)

**Comment by Alexustas** [comment]
![bandicam 2015-10-12 21-27-58-569](https://cloud.githubusercontent.com/assets/6713671/10431697/50c1b166-7128-11e5-9104-7a190adc7fcd.jpg)

**Comment by DeputyLOL** [comment]
ok.... never seen that before.

**Comment by Alexustas** [comment]
ok. MIP Display prop is ready.

![screenshot324](https://cloud.githubusercontent.com/assets/6713671/10440819/5987ccae-7164-11e5-9a27-111c3d98579c.png)

**Comment by Alexustas** [comment]
DL: https://www.dropbox.com/s/pl19lzw1fy4jh4u/ASET_MIP_Display.zip?dl=0

**Comment by DeputyLOL** [comment]
thank you, testing now. Hopefully this screen turns off correctly!

**Comment by police0572** [comment]
It's so beautifull! the creation of great! I want to build boeing777 simulator cockpit, but i don't know the circuit. really envy you!

**Comment by DeputyLOL** [comment]
if you want i can give you some pointers?

**Comment by police0572** [comment]
Thank you for your help, i'm a beginner and i don't know how to build,  can you use arduino nano to achieve？

**Comment by DeputyLOL** [comment]
?

**Comment by Alexustas** [comment]
What else is needed from me as part of this request?

**Comment by DeputyLOL** [comment]
The Control panel in the first picture would probably be needed.
Also an upside-down version of the monitor would be useful for the lower display.

**Comment by DeputyLOL** [comment]
I'm not entirely sure what I'm going to use all of the control panel for yet.

**Comment by Alexustas** [comment]
> Also an upside-down version of the monitor would be useful for the lower display.

add to your monitor config the following module and you will get flipped screen :)

```
MODULE 
{
    name = JSIVariableAnimator
    refreshRate = 10

    VARIABLESET
    {
        controlledTransform  = ScreenObj
        localRotationStart = 0,0,0
        localRotationEnd   = 0,180,0
        variableName = 1
        scale = 0,1
    }
}
```

**Comment by DeputyLOL** [comment]
thank you!

**Comment by Alexustas** [comment]
> The Control panel in the first picture would probably be needed.

from what I see on the panel right now we dont have only a little black buttons

**Comment by DeputyLOL** [comment]
yeah, for now I think all we need is the little black buttons.

**Comment by Alexustas** [comment]
i need to know all labels for that buttons

**Comment by DeputyLOL** [comment]
Until I can throw together some sort of navigation display, those buttons don't really have a purpose.

**Comment by DeputyLOL** [comment]
add to your monitor config the following module and you will get flipped screen :)

Doesn't seem to work.... :(

**Comment by Alexustas** [comment]
![2015-11-23 21-34-25-600](https://cloud.githubusercontent.com/assets/6713671/11341883/8ea0e502-922e-11e5-844b-4b4d231b057a.jpg)

**Comment by Alexustas** [comment]
I tested it before post it

**Comment by DeputyLOL** [comment]
Works now. I thought that the module goes OUTSIDE the prop config like the 'screen brightness' module does.
Apparently not. Thank you.

**Comment by Alexustas** [comment]
> Until I can throw together some sort of navigation display, those buttons don't really have a purpose.

then this request will be is frozen until you decide the details?

**Comment by DeputyLOL** [comment]
yeah, I'm sort of waiting on MOADdv. I have a rough idea of what I need but RPM cant support it and with how busy he is right now, the features I need are probably not coming soon.

The basics are:
Big rotating compass
Show markers/waypoints/points of interests on map <-- (also waiting for waypoints to be added)
Line that shows direction from aircraft to next waypoint
Line that shows the current flight path of the aircraft

Extra:
Show surrounding terrain relative to aircraft height
Show runways.

**Comment by Alexustas** [comment]
so many tasks would be enough for a separate mod :)

**Comment by DeputyLOL** [comment]
yeah I guess. The problem is I have no idea how to make such a thing!
